src/test/java/org/opensearch/ad/transport/handler/AbstractIndexHandlerTest.java

```diff
@@ -73,6 +74,9 @@ enum IndexCreation {
     @Mock
     protected IndexNameExpressionResolver indexNameResolver;
 
+    @Mock
+    protected OpenSearchAsyncClient openSearchAsyncClient;
```

Comment:
Here and above. I know it doesn't align with the variable naming but the wrapper is actually from java client which we created in SDK and for better understanding.
```suggestion
    protected OpenSearchAsyncClient javaAsyncClient;
```